pandas/tests/test_format.py

```diff
@@ -2952,6 +2952,16 @@ def test_to_csv_date_format(self):
         self.assertEqual(df_day.to_csv(), expected_default_day)
         self.assertEqual(df_day.to_csv(date_format='%Y-%m-%d'), expected_default_day)
 
+        # GH 7791
```

Comment:
Is that OK? I mean, in France we always and only use ',' as a decimal separator. It may happen that I'd like to export in CSV a dataframe where, for some reason, I used a float column as index...